src/ASM/LR/ASMu2D.C

```diff
@@ -970,6 +970,14 @@ bool ASMu2D::getElementCoordinates (Matrix& X, int iel) const
 }
 
 
+Vec3 ASMu2D::point (double u, double v, double, int iEl) const
+{
+  Go::Point X0;
+  lrspline->point(X0,u,v,iEl);
+  return SplineUtils::toVec3(X0,nsd);
+}
```

Comment:
this will need an overload in ASMu2Dnurbs. should go through computeBasis in that case to handle the weighting.

Reply:
replaced by a new evalPoint method which uses evaluateBasis() and the element node coordinates to calculate the point coordinates.